src/pages/Home.js

```diff
@@ -215,7 +215,12 @@ class Home extends Component {
                       a({
                         href: 'https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/#:~:text=Machine%20Readable%20Consent%20Guidance&text=Machine%20readable%20consent%20language%20is,to%20for%20their%20research%20purposes',
                         target: '_blank'
-                      }, ['Machine Readable Consent Guidance.'])
+                      }, ['Machine Readable Consent Guidance.']),
+                      " For help determining your data's permitted uses, try our ",
+                      a({
+                        href: '/data_sharing_language_tool',
+                      }, ['Data Sharing Language Tool']),
```

Comment:
This should be a `Link` component.

Reply:
So I just changed it to a Link component and I can't seem to get the link to work, suggestions?